client/packages/requisitions/src/ResponseRequisition/api/hooks/utils/useResponseDisabledForAuthorisation.ts

```diff
@@ -0,0 +1,17 @@
+import { useResponse } from '../document/useResponse';
+import { RequisitionNodeApprovalStatus } from '@common/types';
+import { useIsRemoteAuthorisation } from './useIsRemoteAuthorisation';
+
+export const useResponseDisabledForAuthorisation = (): boolean => {
```

Comment:
:nit I've tried to use this, and typed `useIsResponseDisabledForAuthorisation`
the name is pretty chunky already.. how about `useIsDisabledByAuthorisation` ?
I like having the 'is' in the name